### PR TITLE
Linkable chips

### PR DESCRIPTION
**Pull request by makeitrein**
Screen Shot 2023-01-12 at 11 15 27 AM

Screen Shot 2023-01-12 at 11 15 29 AM

Screen Shot 2023-01-12 at 11 15 32 AM

### CODE REVIEW

File: apps/web/modules/components/entity-table/editable-entity-table-cell.tsx [hunk 1]
```diff
@@ -117,8 +117,12 @@ export const EditableEntityTableCell = ({ cell, space, entityId }: Props) => {
 
         return (
           <div key={`entity-${triple.value.id}`}>
-            <ChipButton icon="check-close" onClick={() => removeOrResetEntityTriple(triple)}>
-              {triple.value.name || triple.value.id}
+            <ChipButton
+              href={NavUtils.toEntity(space, triple.value.id)}
+              icon="check-close"
+              onClick={() => removeOrResetEntityTriple(triple)}
```
Comment (baiirun) [inline apps/web/modules/components/entity-table/editable-entity-table-cell.tsx:123]: Can we rename this to onDeleteClick or something different to differentiate it from clicking the entire chip?

**Review by baiirun — CHANGES_REQUESTED** [review]

File: apps/web/modules/components/entity-table/editable-entity-table-cell.tsx (hunk 1 above)
Comment (makeitrein) [reply apps/web/modules/components/entity-table/editable-entity-table-cell.tsx:123]: Yup, worth just renaming this component to DeleteChipButton? ChipButton not currently being used in any other context

Comment (baiirun) [reply apps/web/modules/components/entity-table/editable-entity-table-cell.tsx:123]: Sounds good to me. Or EditableChipButton or something. I like DeletableChipButton.

File: apps/web/modules/design-system/chip.tsx [hunk 2]
```diff
@@ -46,19 +46,29 @@ const StyledChipButton = styled(StyledChip)(props => ({
...
-    <StyledChipButton as="button" onClick={onClick} role="button">
-      {children}
+    <StyledChipButton as="button" role="button">
+      <Link href={href} passHref>
+        <StyledLink>{children}</StyledLink>
+      </Link>
       {icon === 'check-close' ? (
         // Wrapper div to prevent the icon from being scaled by flexbox
-        <div>
+        <div onClick={onClick}>
```
[the remark this answers is not in the document]
Comment (makeitrein) [reply apps/web/modules/design-system/chip.tsx]: Updated!

**Review by baiirun — APPROVED** [review]
🚀

**Pull request merged**